examples/basic/turbo.json

```diff
@@ -1,5 +1,6 @@
 {
   "$schema": "https://turbo.build/schema.json",
+  "globalDependencies": ["**/.env.*local"],
```

Comment:
@mehulkar
Hi
I found that this setting seems to slow down tasks very much.

Tested with turbo 1.9.6 with basic example:

```
pnpm dlx create-turbo@1.9.6
? Where would you like to create your turborepo? ./my-turborepo
? Which package manager do you want to use? pnpm
```

Then, run `pnpm lint` (with cache == full turbo)
It takes 300-400ms.

```shell
pnpm lint

(omit)

 Tasks:    3 successful, 3 total
Cached:    3 cached, 3 total
  Time:    366ms >>> FULL TURBO
```

If I remove the globalDependencies setting:

```shell
pnpm lint

(omit)

 Tasks:    3 successful, 3 total
Cached:    3 cached, 3 total
  Time:    55ms >>> FULL TURBO
```

It is turbo!

If I make the following changes, it is also very fast.
(As much as when I delete globalDependencies.)

```diff
-  "globalDependencies": ["**/.env.*local"],
+  "globalDependencies": ["apps/**/.env.*local"],
```

So I think it may be better to limit the scope of glob.
What do you think?

Reply:
Yes it’s not a great solution for exactly this reason. You can specify all known files yourself or use a more limited glob. 

@nathanhammond is also working on a feature to make this a top level configuration